src/main/java/md/teckwillacademy/gitworkflow/GitWorkFlow.java

```diff
@@ -0,0 +1,37 @@
+package md.teckwillacademy.gitworkflow;
+
+public class GitWorkFlow {
+    public static void main(String[] args) {
+        String step1 = "Execute - git stash command in order to save the changes";
+        String step2 = "Go to the main branch - git checkout master";
+        String step3 = "Get the latest changes - sync with Origin/master - git fetch";
+        String step4 = "Download the diffs - git pull";
+        String step5 = "Create a new dedicated branch for the task - git checkout -b Branch_name";
+        String step6 = "Add code changes for implementing the required task";
+        String step7 = "Save the changes on local memory - git commit";
+        String step8 = "Upload the changes on the cloud - git push";
+        String step9 = "Go to the GitHub repository";
+        String step10 = "Create a new Pull Request into the origin/master branch";
+        String step11 = "Invite collaborators to review the code";
+        String step12 = "Merge the code";
```

Comment:
Thank you Kate, please add the following steps:

String step_12_1 = "If changes are required then I have to modify my local branch and perform a git commit amend";
String step_12_2 = "After git commit amend, we have to do a git push -f ";

System.out.println(step_12_1);
System.out.println(step_12_2);